src/behavior/src/A32NX_Interior_Generics.xml

```diff
@@ -644,4 +644,95 @@
         </Component>
     </Template>
 
+
+    <!--
+    Template for syncing lights. Works by simultaneously toggling the light circuit buses temporary, given a condition.
+    Requires the lights in *.fx to be synced and timed properly to work, as this template only ensures a common timeline.
+
+    Main Parameters:
+            - SYNC_FREQUENCY        Default: 1          Speed of update loop, N per second.
+            - SYNC_TOGGLE_BUSES     <no default>        The buses for the lights that should be synced.
+            - SYNC_CONDITION        <no default>        The condition to trigger a sync. This is often two or more switches.
+            - SYNC_REQUISITE        <no default>        The required main buses for the circuits.
+    -->
+    <Template Name="FBW_Light_Sync">
+        <DefaultTemplateParameters>
+            <SYNC_FREQUENCY>1</SYNC_FREQUENCY>
+            <SYNC_TOGGLE_BUSES></SYNC_TOGGLE_BUSES>
+            <SYNC_CONDITION></SYNC_CONDITION>
+        </DefaultTemplateParameters>
+
+        <UseTemplate Name="ASOBO_GT_Update">
+            <FREQUENCY>#SYNC_FREQUENCY#</FREQUENCY>
+            <UPDATE_CODE>
+                #SYNC_REQUISITE# if{
+                    #SYNC_CONDITION# if{
```

Comment:
It would be cool to do this in RNP... but there are footguns with including code like this so RPN is not so bad here.

Reply:
Footguns sounds dangerous and risky. This is not the RPN-conversion you are looking for. :neckbeard: